lib/embed/embed.dart

```diff
@@ -54,6 +56,10 @@ class PlaygroundMobile {
   Gist backupGist;
   Router _router;
 
+  Document _dartDoc;
```

Comment:
Are these three fields referenced? I can't see where they're used in the source.